Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -73,37 +84,56 @@ gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.getRendererObject = function(
   return this._pixiObject;
 };
 
+// REVIEW comparé le wrap avec celui de l'IDE
 gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateWordWrap = function() {
   this._pixiObject.style.wordWrap = this._object._wordWrap;
   this._pixiObject.dirty = true;
   this.updatePosition();
 };
 
+// REVIEW comparé le wrap avec celui de l'IDE
 gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateWrappingWidth = function() {
-  this._pixiObject.maxWidth = this._object._wrappingWidth;
+  if (this._object.wordWrap) {
+    this._pixiObject.maxWidth = width;
+  } else {
+    this._pixiObject.maxWidth = 0;
+  }
+
+    
+
+
   this._pixiObject.dirty = true;
+
+ 
+  this._pixiObject.maxWidth = this._object._wrappingWidth;
```

Comment:
Seems like you're doing some redundant work here (setting twice `this._pixiObject.maxWidth`), something to double check I guess :)